pkg/runit/servicebuilder.go

```diff
@@ -74,6 +74,7 @@ var DefaultBuilder = &ServiceBuilder{
 }
 
 var DefaultChpst = "/usr/bin/chpst"
+var DefaultContain = "/usr/bin/contain"
```

Comment:
we don't actually have a DefaultNolimit right now, so what package did you have in mind? Personally I want to take all three of these things (nolimit, chpst, contain) and factor them into something "prettier", but I don't know what it should look like right now.